src/send_command.rs

```diff
@@ -191,7 +202,7 @@ impl SendCommandState {
     pub fn set_authenticate_data(
         &mut self,
         authenticate_data: AuthenticateData<'static>,
-    ) -> Result<ClientFlowCommandHandle, AuthenticateData> {
+    ) -> Result<ClientFlowCommandHandle, AuthenticateData<'static>> {
```

Comment:
Should I make a small PR for fixing this on the main branch?

Reply:
Sure, it will reduce noise on PRs. Let me know when it's done.

Reply:
Done #166